src/world.js

```diff
@@ -52,8 +52,11 @@ $.World = function( options ) {
     this.viewer = options.viewer;
     this._items = [];
     this._needsDraw = false;
+    this._autoRefigureSizes = true;
     this._delegatedFigureSizes = function(event) {
-        _this._figureSizes();
+        if (this._autoRefigureSizes) {
```

Comment:
Hmm...actually, it looks like this is the source of the broken unit test as well.